CONTRIBUTING.md

```diff
@@ -2,32 +2,55 @@
 
 First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODE_OF_CONDUCT.md#code-of-conduct).
 
-### Report a bug
+### To report a bug
 
 * run `brew update` (twice)
 * run and read `brew doctor`
 * read [the Troubleshooting Checklist](https://docs.brew.sh/Troubleshooting)
 * open an issue on the formula's repository
 
-### Submit a version upgrade for the `foo` formula
+### To submit a version upgrade for the `foo` formula
 
 * check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).
 * `brew bump-formula-pr --strict foo` with `--url=...` and `--sha256=...` or `--tag=...` and `--revision=...` arguments.
 
-### Add a new formula for `foo` version `2.3.4` from `$URL`
+### To add a new formula for `foo` version `2.3.4` from `$URL`
 
 * read [the Formula Cookbook](https://docs.brew.sh/Formula-Cookbook) or: `brew create $URL` and make edits
 * `brew install --build-from-source foo`
 * `brew audit --new-formula foo`
 * `git commit` with message formatted `foo 2.3.4 (new formula)`
 * [open a pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) and fix any failing tests
 
-### Contribute a fix to the `foo` formula
-
-* `brew edit foo` and make edits
-* leave the [`bottle`](http://www.rubydoc.info/github/Homebrew/brew/master/Formula#bottle-class_method) as-is
-* `brew uninstall --force foo`, `brew install --build-from-source foo`, `brew test foo`, and `brew audit --strict foo`
-* `git commit` with message formatted `foo: <insert details>`
-* [open a pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) and fix any failing tests
+### To contribute a fix to the `foo` formula
+
+If you are already well versed in the use of `git`, then you can find the local
+copy of the `homebrew-core` repository in this directory
+(`$(brew --prefix)/Homebrew/Library/Taps/homebrew/homebrew-core/`), modify the formula there
+leaving the section `bottle do ... end` unchanged, and prepare a pull request
+as you usually do.  Before submitting your pull request, be sure to test it
+with these commands:
+
+```
+brew uninstall --force foo
+brew install --build-from-source foo
+brew test foo
+brew audit --strict foo
+```
+
+After testing, if you think it is needed to force the corresponding bottles to be
+rebuilt and redistributed, add a line of the form `revision 1` to the formula,
+or add 1 to the revision number already present.
+
+If you are not already well versed in the use of `git`, then you may learn
+about it from the introduction at
+https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request and then proceed as
+follows:
+
+* run `brew edit foo` and make edits
+* leave the section `bottle do ... end` unchanged
+* test your changes using the commands listed above
+* run `git commit` with message formatted `foo <insert details>`
```

Comment:
```suggestion
* run `git commit` with message formatted `foo: <insert details>`
```

Reply:
I just checked again and the bot does use it. Sorry about that, I guess I have to change my own habbit too then 😅.